src/test/java/com/meilisearch/integration/ClientTest.java

```diff
@@ -28,10 +30,6 @@ public void initialize() {
         setUp();
         setUpJacksonClient();
         if (testData == null) testData = this.getTestData(MOVIES_INDEX, Movie.class);
-    }
-
-    @AfterAll
-    static void cleanMeilisearch() {
         cleanup();
     }
```

Comment:
I think you can also keep the `afterAll` otherwise the last index of the last test will not be deleted